src/styles/creation.scss

```diff
@@ -0,0 +1,43 @@
+.back-button {
```

Comment:
You don't need to put button styling in every scss page, you can put it in the global page. It will be applied across the entire project.